io/common/virtual_bind_unbind.py.data/README.txt

```diff
@@ -0,0 +1,12 @@
+This script will unbind and bind the virtualized device
+for vnic, vscsi, vfc,veth as many times as specified by 
+the user
+
+Inputs Needed :
+------------------------------------
+virtual_device - virtual device for the test interface or disk name.
+virtual_slot - slot name for the virtual device, which can get it from lscfg -vl command.
```

Comment:
also provide example slot name in README like
virtual_slot: U9009.42A.13C6F1W-V2-C201

Reply:
ok